Documentation/ceph-csi-drivers.md

```diff
@@ -21,14 +21,11 @@ For documentation on consuming the storage:
 
 ## RBD Snapshots
 
-Since this feature is still in [alpha
-stage](https://kubernetes.io/blog/2018/10/09/introducing-volume-snapshot-alpha-for-kubernetes/)
-(k8s 1.12+), make sure to enable the `VolumeSnapshotDataSource` feature gate on
-your Kubernetes cluster API server.
+Requires kubernetes v1.17+ which supports snapshot beta.
 
-```console
---feature-gates=VolumeSnapshotDataSource=true
-```
+**Note**: Its kubernetes distributor responsibility to install new snapshot
```

Comment:
Instead of a "note", how about adding a subheading for prerequisites? They would include 1) k8s 1.17+ and 2) the snapshot controller to be installed.